.env.example

```
@@ -222,6 +222,28 @@ APP_KEY=
+#   1. Create a Fly.io account: https://fly.io
+#   2. Create an API token: fly tokens create org
+#   3. Set COMPUTE_SERVICES_ENABLED=true
+#   4. Set FLY_API_TOKEN to your token
+#
+# Optional: Set a custom domain for service URLs (default: compute.insforge.dev)
+#   - Point *.compute.yourdomain.com to Fly.io via DNS
+
+COMPUTE_SERVICES_ENABLED=false
+FLY_API_TOKEN=
+FLY_ORG=insforge
+COMPUTE_DOMAIN=compute.insforge.dev
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The COMPUTE_DOMAIN key should go before the COMPUTE_SERVICES_ENABLED key

(UnorderedKey)